lib/models/hcktest_config.rb

```diff
@@ -12,12 +12,21 @@ class Parameter < T::Struct
       const :value, String
     end
 
+    class Command < T::Struct
+      extend T::Sig
+
+      const :guest, T.nilable(String)
```

Comment:
I suggest to add some description and log it before run, to make logs more informative.